Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1061,6 +1063,25 @@ public boolean initSessionForced(BranchReferralInitListener callback) {
         return false;
     }
 
+    /**
+     * <p>Force initialises a session with the Branch API with bypassing the intentState_ READY callback from the current Activity, assigning a {@link BranchReferralInitListener}
+     * to perform an action upon successful initialisation.</p>
+     *
+     * @param callback A {@link BranchReferralInitListener} instance that will be called following
+     *                 successful (or unsuccessful) initialisation of the session with the Branch API.
+     * @return A {@link Boolean} value, indicating <i>false</i> if initialisation is
+     * unsuccessful.
+     */
+
+    public boolean initSessionWithoutState(BranchReferralInitListener callback) {
```

Comment:
@code-crusher can we find a way to get rid of this and just use the boolean `bypassCurrentActivityIntentState_`?